mlflow/sagemaker/__init__.py

```diff
@@ -119,7 +119,8 @@ def push_image_to_ecr(image=DEFAULT_IMAGE_NAME):
     account = caller_id['Account']
     my_session = boto3.session.Session()
     region = my_session.region_name or "us-west-2"
-    fullname = _full_template.format(account=account, region=region, image=image)
+    fullname = _full_template.format(account=account, region=region, image=image, version=version)
+    print("Pushing docker image {image} to {repo}".format(image=image, repo=fullname))
```

Comment:
I think the current convention is to use eprint for informative printouts (mlflow.utils.logging_utils).